metadata/get_dataset_features.py

```diff
@@ -0,0 +1,211 @@
+import argparse
+import multiprocessing
+import pyspark
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    ArrayType,
+    IntegerType,
+    StringType,
+)
+import pyspark.sql.functions as F
+from typing import Callable
+import pandas as pd
+from tqdm import tqdm
+import logging
+import stanza
+import sys
+from transformers import AutoTokenizer
+import time
+from collections import defaultdict
+
+sys.path.append("../dolma_data_processing/")
```

Comment:
I'm not sure we need this (also not sure how this file is different from the one in `dolma_data_processing`?). We have an init.py in both directories so we can also just do `from dolma_data_processing.calc_feature_utils import *` and call with `python -m metadata.get_dataset_features <...>`

Reply:
Yup, I guess re: earlier comment on what was different, we could change this. It's up to you, I thought it would be cleaner to keep it in this dir though since it's no longer dolma specific. Let me know what you prefer.

Reply:
yeah, I guess technically nothing in `dolma_data_processing` is super dolma specific? Bit of a small point though so we can also just keep this file here for now